css/style.css

```diff
@@ -388,3 +424,213 @@ hr.works-divider {
   border-radius: 10px;
   height: 5px;
 }
+
+.lg-none {
+  display: flex;
+}
+
+.sm-none {
+  display: none;
+}
+
+@media screen and (min-device-width: 768px) {
+  .sm-none {
+    display: flex;
+  }
+
+  .header-container {
+    height: 72px;
+    padding-inline: 12%;
+  }
+
+  .headline-container {
+    background: #1c1a19 url("../images/Illustration-header.png") 100% 0% no-repeat;
+    background-size: 650px auto;
+    height: 92vh;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 88px;
+    padding: 0 36px;
+  }
+
+  .headline-content {
+    height: 739px;
+    width: 33%;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    order: 1;
+    padding: 192px 0;
+  }
+
+  .headline-title {
+    font-size: 72px;
+    line-height: 81.5px;
+  }
+
+  .headline-subtitle {
+    font-size: 32px;
+    line-height: 44px;
+    font-weight: 500;
+  }
+
+  .headline-text {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .social-media-button-group {
+    flex-direction: column;
+    justify-content: center;
+    width: auto;
+    padding: 0;
+  }
+
+  .works-container {
+    display: grid;
+    grid-template-columns: repeat(3, 33%);
+    gap: 24px;
+    padding-inline: 10%;
+    padding-block: 120px;
+  }
+
+  .works-title-container {
+    grid-column: 1/4;
+    flex-direction: row;
+    margin-bottom: 90px;
+  }
+
+  .works-title {
+    margin: 0;
+  }
+
+  .works-divider {
+    width: 70%;
+  }
+
+  .work-main {
+    grid-column: 1/4;
+    flex-direction: row;
+    gap: 24px;
+  }
+
+  .work-main-image-container {
+    text-align: center;
+    flex: 2;
+  }
+
+  .work-container {
+    box-sizing: border-box;
+    height: 435px;
+  }
+
+  .bg-image {
+    padding: 16px;
+    justify-content: flex-end;
+  }
+
+  .work-light {
+    background-image: url('../images/Image-ProfesionalPrint.png');
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end;
+  }
+
+  .work-title.light {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .work-text.light {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .data-image {
```

Comment:
- [ ] Kindly make sure the **Hamburger** button is visible and **all the navigations options(e.g.: Portfolio, Contact,...)** are hidden on the **"Mobile Version"**. And on the Desktop version, the **Hamburger** button should be hidden, and **all the navigation options** are visible. Right now, your navigations options are visible for both the **Mobile Version** and **Desktop Vesrion**